tests/integration/security/authz_test.go

```diff
@@ -967,8 +967,13 @@ func TestAuthz_PathPrecedence(t *testing.T) {
 }
 
 func TestAuthz_PathTemplating(t *testing.T) {
+	minIstioVersion := "1.22.0"
```

Comment:
should this be a constant value or do we expect it to change? Also, is the min. istio version specific to just authz path templating?

Reply:
> should this be a constant value or do we expect it to change? 
- This will be a constant value since path templating support is added from v1.22. Not expected to change thus defined inside the function.

> Also, is the min. istio version specific to just authz  path templating?
- Correct, this min version is test specific

Reply:
Hi @MorrisLaw, Just following up if this is fine. I would like to merge this as early as possible if everything looks fine.